infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/utils/LogReplicationConfigManager.java

```diff
@@ -67,49 +77,161 @@ public class LogReplicationConfigManager {
 
     private Table<VersionString, Version, Uuid> pluginVersionTable;
 
+    // In memory entries of registry table, which will be refreshed when config is syncing with registry table.
+    // This is needed to avoid inconsistency between config fields (for example, after streamsToReplicate is fetched,
+    // some tables opened and confirmedNoisyStreams will be read with potential conflict info)
+    private List<Map.Entry<TableName, CorfuRecord<TableDescriptors, TableMetadata>>> entryList;
```

Comment:
nit - this can be renamed to something more explanatory, such as registryTableEntries